src/app/globals.css

```
@@ -106,3 +61,227 @@ html {
+  --chart-3: oklch(0.72 0 0);
+  --chart-4: oklch(0.92 0 0);
+  --chart-5: oklch(0.56 0 0);
+  --sidebar: oklch(0.99 0 0);
+  --sidebar-foreground: oklch(0 0 0);
+  --sidebar-primary: oklch(0.4735 0.2068 293.0625);
+  --sidebar-primary-foreground: oklch(1 0 0);
+  --sidebar-accent: oklch(0.94 0 0);
+  --sidebar-accent-foreground: oklch(0 0 0);
+  --sidebar-border: oklch(0.92 0 0);
+  --sidebar-ring: oklch(0 0 0);
+  --font-sans: Montserrat, ui-sans-serif, sans-serif, system-ui;
```

Stylelint (17.7.0)

[error] Expected "Montserrat" to be "montserrat" (value-keyword-case)

(value-keyword-case)